net/netfilter/nf_tables_api.c

```diff
@@ -28,6 +28,9 @@
 
 #define NFT_MODULE_AUTOLOAD_LIMIT (MODULE_NAME_LEN - sizeof("nft-expr-255-"))
 
+unsigned int nf_tables_net_id __read_mostly;
+EXPORT_SYMBOL_GPL(nf_tables_net_id);
```

Comment:
I suspect this `EXPORT_SYMBOL_GPL` is a RedHat thing, i haven't seen it in the community, it was there in the introduction and its not there now.

Reply:
It's  in 534?  I'll leave it for now since matching Red Hat is what we're doing, rather than matching upstream.  If you disagree we can reconsider.

Reply:
Yes, Sorry there was some wording issues with my first response.  I don't see this EXPORT in the community at all but I see it in `4.18.0-534`

https://github.com/ctrliq/kernel-src-tree/blob/centos_kernel-4.18.0-534.el8/net/netfilter/nf_tables_api.c#L33